financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 
+transparent and unambiguous information in funding requests. 
+This template is not intended to be exhaustive and only serves as a guideline to clearly communicate the minimal amount of information 
+needed for the GRIN community to assess requests for funding. Feel free to add and modify to this template to improve clarity and transparency. 
+However, let us aim to keep this funding template minimal and elegant like GRIN and the Mimble Wimble protocol itself. 
+For ease of comparison of funding requests, it is advised to provide these fields of information in the order given below.
+
+## Financial information
+�	Funding period: 
+o	(e.g. 3 months)
+�	Total amount of requested funding:
+o	State total amount of requested funding in this period as well as in the preceding funding period both in USD and local currency

```

Comment:
1. We've been discussing currency use in the [spending guidelines RFC](https://github.com/mimblewimble/grin-rfcs/pull/41) and in [meetings](https://github.com/mimblewimble/grin-pm/blob/master/notes/20200310-meeting-governance.md#42-general-fund-spending-guidelines). 

   We concluded that we're happy to have funding requests being denominated in any "high profile fiat currency". It's a good thing to provide a rough conversion to USD for comparison, but it's the local currency that will be driving the request, so that probably should be made clear in the template. Example:
   > I wish to request GBP 100.00 in funding (~ USD 123.00 as of April 6)

2. It's not certain that there will be a preceding funding period, so might make sense to have that as optional:
   > If this is part of a series of funding requests, please state whether the amount or currency has changed since the last request, and if so how.